### DISCUSSION TITLE
data rate is prebuilt mod in vp plus? or part of docker package here?

### DISCUSSION THREAD

**Discussion by jagat95** [lloesche/valheim-server-docker#339] [Q&A]
Hi,

I tried enabling VP Plus to increase data send rate as suggested in docs but not able to feel any difference.
- How do I identify its working?
- Do I need to install any mod along with VP Plus?

**Comment by lloesche** [comment]
When you press `F2` in-game you should see the data send and received rate. When the mod is inactive it'll never go above 64 kb/s. When applying the mod you'll see it jump to 200+ kb/s.

**Reply by jagat95** [reply]
any idea if this is still applicable? as I read in few posts that some mods were using specific configvalue edit , but it was removed by developers. So do you know if this VPPlus mod still uses it or not, as I checked documentation that configuration isnt available there.

So just doubtful if this needs to be kept in this docker img .cfg still or not
